civictechprojects/static/css/partials/_Profile.scss

```diff
@@ -114,6 +114,34 @@
   }
 }
 
+// for use with IframeGroupDisplay to make it full size in iframe
+.frame-full {
+  margin: 0;
+  width: 100%!important; // important to override the media queries
+}
+
+.Profile-primary-container.frame-full .AboutGroup-card-container .row .ProjectCardContainer .row{
+  margin: 0;
+  row-gap: 1rem;
+  column-gap: 1rem;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 100vh; // force the flex box to use the whole iframe height that it is in
+}
+
+.Profile-primary-container.frame-full .AboutGroup-card-container .row .ProjectCardContainer .row .col-sm-12.col-lg-6 .ProjectCard-root {
+  margin: 0;
+}
+
+// for the project cards burried in the iframe, give them a fixed width and let them fill the frame
+.Profile-primary-container.frame-full .AboutGroup-card-container .row .ProjectCardContainer .row .col-sm-12.col-lg-6 {
+  flex: 0 0 26rem;
```

Comment:
Declaring a fixed width here is causing some issues, most obviously on any device where the iframe is below 26rem width - like, say, a mobile device. It also makes cards look unusual at very high res, so a more nuanced solution is needed here - perhaps with max-width, media queries? If it's not working on mobile and it looks ... real odd on high res, let's give it another pass?

Reply:
Okay, I found the Google Pixel Fold with a width of 280px that broke with 26rem.  As you suggested, I applied it in a breakpoint above small
<img width="172" alt="image" src="https://github.com/DemocracyLab/CivicTechExchange/assets/3317487/e680aeb7-ea53-4099-a0b5-19064d83bcd0">

As for the high res case, the parent web site will have their own breakpoints and we should avoid all the interactions.  In the open san diego case their content is contained in 960px container.  And I've seen similar containers on other brigade sites.   I suggest we avoid spending too much time perfecting how we render when it's really wide for an outlier use case, and focus on getting this feature out so that Mark can start pitching it to orgs.  After that, I expect we will get some real end-user feedback.   -- but I did fix the project display in the wide case - see that comment.